### ISSUE TITLE
Maintainer on vacation from Jan 5 through Jan 16, 2024

### ISSUE THREAD

**Issue by mdmintz** [seleniumbase/SeleniumBase#2415]
Hey, this is the maintainer of **SeleniumBase**. I'll be on vacation Jan 5 through Jan 16. Responses will be delayed.
I might respond to simple questions via smartphone, but I plan on keeping the laptop lid closed during the trip.

Here are some things to keep you busy while I'm away:
* [The SeleniumBase Playlist on YouTube](https://www.youtube.com/playlist?list=PLp9uKicxkBc5UIlGi2BuE3aWC7JyXpD3m)
* [My YouTube Channel with SeleniumBase videos](https://www.youtube.com/@MichaelMintz/videos) - Over 500 subscribers!
* [SeleniumBase GitHub Docs](https://github.com/seleniumbase/SeleniumBase) - (Technically you're already here.)

2023 was a big year. Here are my open source contributions from the past 12 months:

<a href="https://github.com/mdmintz?tab=overview&from=2023-12-01&to=2023-12-31"><img width="780" alt="Screenshot 2024-01-04 at 8 19 55 PM" src="https://github.com/seleniumbase/SeleniumBase/assets/6788579/bdb2f600-d9ad-4797-80bd-7521698d61da"></a>

It's now 2024, with lots of things coming. I hope to keep the momentum going with more YouTube videos, etc.

----

By far, UC Mode has been the most popular SeleniumBase feature recently, which became even more popular after the UC Mode video tutorial on YouTube reached over 5000 views in the first few months:

<b>https://www.youtube.com/watch?v=5dMFI3e85ig</b>

<a href="https://www.youtube.com/watch?v=5dMFI3e85ig"><img src="https://github.com/seleniumbase/SeleniumBase/assets/6788579/4c9a112a-25c6-458f-83c6-c4b910b8b636" title="undetected automation" width="330" /></a>

### Here are the existing examples that use UC Mode:
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/verify_undetected.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/uc_cdp_events.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_uc_mode.py

### If you need clicks to go undetected, try the UC Mode examples that click undetected:
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_turnstile.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_form_turnstile.py

----

Let's not forget that SeleniumBase is ultimately a test automation framework (even though lots of its recent popularity comes from bypassing captchas in UC Mode). I'll work on better marketing for the test automation side of things, as SeleniumBase can easily compete head-to-head with the best test automation frameworks out there.

Star SeleniumBase on GitHub ⭐ [Thanks to everyone who already gave us a Star!](https://github.com/seleniumbase/SeleniumBase/stargazers) 🎉

**Comment by mdmintz** [comment]
I'm back from vacation!

My return flight was very delayed and arrived in the middle of the night, so I'm a bit sleep-deprived today and not yet at full power... but will be soon!

<img width="420" alt="Screenshot 2024-01-17 at 7 08 52 PM" src="https://github.com/seleniumbase/SeleniumBase/assets/6788579/001b4196-bfbb-4fc0-813a-f8dc3358dbfe">

There's a lot of work to be done and keep me busy. I also started answering some SeleniumBase questions... the ones that I haven't already answered from my phone while away.

--------

Here are some of my photos from traveling "the world":

<img width="520" alt="Screenshot1" src="https://github.com/seleniumbase/SeleniumBase/assets/6788579/926e0a84-906d-403f-8fb8-9c99a996a92b">

<img width="520" alt="Screenshot2" src="https://github.com/seleniumbase/SeleniumBase/assets/6788579/f3a59b3c-dadd-4a84-aa29-519afc8de16e">

<img width="520" alt="Screenshot3" src="https://github.com/seleniumbase/SeleniumBase/assets/6788579/e735b659-4737-43d1-9035-20cbdf4358fd">

<img width="520" alt="Screenshot4" src="https://github.com/seleniumbase/SeleniumBase/assets/6788579/6c6be9a1-7e9c-4eea-81dd-7260d9a10412">

--------

For some trivia, see if you can figure out where I went.

**Comment by mdmintz** [comment]
For those of you playing the Vacation Trivia Game: If you guessed China and/or Japan, then you were close. Those first three photos were taken at Epcot in the China and Japan sections. The dolphin photo was taken at SeaWorld. (All of those are located in Orlando, Florida). Thanks for playing "Vacation Trivia"! (P.S. I also visited other cities on my trip.)

--------

I'm officially back in action because I just shipped a new release: `4.22.6` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.22.6

My next planned trip will be a long weekend for the 2024 Solar Eclipse:
--> https://science.nasa.gov/eclipses/future-eclipses/eclipse-2024/where-when/
